src/highlight/__tests__/HighlightAnnotations-test.tsx

```diff
@@ -30,17 +65,103 @@ describe('components/highlight/HighlightAnnotations', () => {
 
             expect(wrapper.find(HighlightList).exists()).toBe(true);
             expect(wrapper.exists(HighlightCreator)).toBe(false);
+            expect(wrapper.exists('.ba-HighlightAnnotations-target')).toBe(false);
```

Comment:
We typically want to avoid asserting on CSS classes. Can we check if the inner SVG/Canvas components exist, instead?